### PR TITLE
remove kv store support

### PR DESCRIPTION
**Pull request by guits**
the next stable release will drop this feature.

Signed-off-by: Guillaume Abrioux <gabrioux@redhat.com>

### CODE REVIEW

File: roles/ceph-config/tasks/main.yml [hunk 1]
```diff
@@ -135,11 +130,8 @@
     run_once: true
     when:
       - (cephx or generate_fsid)
-      - (not mon_containerized_default_ceph_conf_with_kv and
-          (inventory_hostname in groups.get(mon_group_name, []))) or
-        (not mon_containerized_default_ceph_conf_with_kv and
-          ((groups.get(nfs_group_name, []) | length > 0)
-            and (inventory_hostname == groups.get(nfs_group_name, [])[0])))
+      - ((inventory_hostname in groups.get(mon_group_name, [])) or
+        (groups.get(nfs_group_name, []) | length > 0))
```
Comment (leseb) [inline roles/ceph-config/tasks/main.yml]: what about the `and (inventory_hostname == groups.get(nfs_group_name, [])[0]`?

Comment (guits) [reply roles/ceph-config/tasks/main.yml]: @leseb should we simply remove it?

**Comment by leseb** [comment]
jenkins test pipeline

**Comment by guits** [comment]
jenkins test pipeline

**Comment by guits** [comment]
jenkins test pipeline

File: roles/ceph-container-common/tasks/main.yml [hunk 2]
```diff
@@ -17,10 +17,9 @@
 - name: include checks.yml
   include_tasks: checks.yml
   when:
-    - (not containerized_deployment_with_kv and
-       ((inventory_hostname in groups.get(mon_group_name, [])) or
-        (inventory_hostname in groups.get(nfs_group_name, [])) or
-        (inventory_hostname in groups.get(osd_group_name, []))))
+    - (((inventory_hostname in groups.get(mon_group_name, [])) or
+      (inventory_hostname in groups.get(nfs_group_name, [])) or
```
Comment (leseb) [inline roles/ceph-container-common/tasks/main.yml]:
```suggestion
      ((inventory_hostname in groups.get(nfs_group_name, [])) or
```

File: roles/ceph-container-common/tasks/main.yml [hunk 3]
```diff
@@ -17,10 +17,9 @@
...
+      (inventory_hostname in groups.get(osd_group_name, [])))
```
Comment (leseb) [inline roles/ceph-container-common/tasks/main.yml]:
```suggestion
      ((inventory_hostname in groups.get(osd_group_name, [])))
```

File: roles/ceph-container-common/tasks/main.yml [hunk 4]
```diff
@@ -21,10 +21,9 @@
 - name: include checks.yml
   include_tasks: checks.yml
   when:
-    - (not containerized_deployment_with_kv and
-       ((inventory_hostname in groups.get(mon_group_name, [])) or
-        (inventory_hostname in groups.get(nfs_group_name, [])) or
-        (inventory_hostname in groups.get(osd_group_name, []))))
+    - ((inventory_hostname in groups.get(mon_group_name, []) or
```
Comment (leseb) [inline roles/ceph-container-common/tasks/main.yml]:
```suggestion
    - (inventory_hostname in groups.get(mon_group_name, []) or
```

**Comment by guits** [comment]
jenkins test dev-centos7_cluster

**Comment by guits** [comment]
jenkins test dev-docker_cluster

**Pull request merged**